src/spender/paths/spender@files@generate_urls@bulk.yaml

```diff
@@ -0,0 +1,68 @@
+post:
+  operationId: files_generate_urls_bulk_post
+  tags:
+    - Files
+  summary: Bulk generate download and upload URLs
+  description: |
+    Once a file is created, its contents may need to be uploaded (or downloaded). To upload (or download) contents, you'll
+    need to generate a temporary signed URL. This API accomplishes that. The signed URL is valid for 5 mins.
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          type: object
+          additionalProperties: False
+          required:
+            - data
+          properties:
+            data:
+              type: array
+              items:
+                $ref: ../../components/schemas/file.yaml#/file_generate_urls_action_in
+              example:
+                - id: sdfd2391
+                - id: sdfd2392
+  responses:
+    '200':
+      description: OK
+      content:
+        application/json:
+          schema:
+            type: object
+            additionalProperties: False
+            properties:
+              data:
+                type: array
+                items:
+                  $ref: ../../components/schemas/file.yaml#/file_generate_urls_action_out
```

Comment:
how will this work?
`file_generate_urls_action_out` doesn't have `thumbnail_200_200_download_url`